layers/+source-control/git/README.org

```diff
@@ -136,6 +159,48 @@ To enable it you have to add the following lines to your
 #+END_SRC
 
 ** Forge
+
+Magit Forge can view and create issues & pull requests with forges
+(e.g. GitHub, GitLab)
+
+Magit Forge requires a the username for the respective forge and will prompt for a
+value if not defined in  =~/.gitconfig=
+Or define your forge identity using the following command in a terminal window
```

Comment:
```suggestion
Magit Forge requires the username for the respective forge and will prompt for a
value if not defined in  =~/.gitconfig=.
Or define your forge identity using the following command in a terminal window.
```

Reply:
removed 'the' from the sentence to correct the grammer